setup_wizard.py

```
@@ -0,0 +1,336 @@
+    """Run the setup wizard"""
+    print_banner()
+    
+    # Check requirements
+    if not check_requirements():
+        print("\n❌ Requirements not met. Please fix the issues above and try again.")
+        sys.exit(1)
+    
+    print("\n✅ All requirements met!")
+    
+    # Set up first flight
+    flight_id = setup_first_flight()
```

Ruff (0.12.2)

Local variable `flight_id` is assigned to but never used

Remove assignment to unused variable `flight_id`

(F841)